grafana/dashboards/cmode/cluster.json

```diff
@@ -1876,6 +2050,7 @@
         },
         {
           "datasource": "${DS_PROMETHEUS}",
+          "description": "System CPU resource utilization. Returns a computed percentage for the default CPU field. Basically computes a 'cpu usage summary' value which indicates how 'busy' the system is based upon the most heavily utilized domain. The idea is to determine the amount of available CPU until we're limited by either a domain maxing out OR we exhaust all available idle CPU cycles, whichever occurs first.",
```

Comment:
change to match cdot dashboard
might be worth pulling this description into a constant var and updating panels to use it